src/triggers/AccountTrigger.trigger

```diff
@@ -0,0 +1,5 @@
+trigger AccountTrigger on Account (before insert, before delete) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid logic in triggers](https://app.codacy.com/manual/rsoesemann/apex-pmd-samples/pullRequest?prid=5116698)